### ISSUE TITLE
HQ Week 1/21 

### ISSUE THREAD

**Issue by ritahitching** [lzim/teampsd#895]
Cross ref #889 
### 1. Manuscripts
- [ ]  Review R01 consort tables on OSF (https://osf.io/8pzxu/wiki/consort_checklist)
- [ ]  Work with Stacey to re-submit VAPOR
- [ ]  Work with Swap to submit Mentor
- [ ]  Work to edit/finalize the graphics for GMB w/Stacey & David
- [ ]  R21 preliminary findings

### 2. MTL National Launch
- [ ]  Review SAY guides & close documentation cards
- [ ]  Bookdown format for the Manual

### 3. Manuscripts 
- [ ]  Review IIR consort tables on OSF for IIR (https://osf.io/p7g8f/wiki/consort_checklist/)
- [ ]  CC Intro & Discussion
- [ ]  MM Intro & Discussion
- [ ]  Psy Intro & Discussion
- [ ]  SP Intro & Discussion

### 4. Other Upcoming
- [ ] 1. NCPTSD website update - who handles this at NCPTSD 

## Delegate to #hq for handling:
- [ ] Email Dan O'Brien Mazza to update on MTL Launch before retirement 
- [ ] Set up local VAPOR meetings with Jill Hudson and Ben Marks, etc.

### Tuesday
- [x] r01 Annual progress report
- [ ] Ping across team to help seed questions for Community of Practice.
- [ ] Review MTL facilitate launch meeting notes from Monday w/ @ritahitching  and @lijenn 

### Wednesday
- [ ] PCSPD to Ella w/ @lijenn  (need info from @lzim)
- [ ] Review MTL facilitate launch meeting notes w/ @ritahitching  and @lijenn 
- [ ] review any SOP questions w/ @lijenn after her review
- [ ] review slides/updates from Savet #674 
- [ ] Inventory for 8am Thurs meeting w/Savet all open items and training/documentation needed to be trained on and reviewed throughout week of 1/20
- [ ] R01 protocol w/ @lzim

### Thursday
- [ ] co-I manuscript review schedule
- [ ] work with @lijenn to submit Rebrandly expense monthly
- [ ] new monthly meeting time with Chris Crowe 
- [ ] new monthly meeting time with Kelly Maieritsch
- [ ] CRM Instance and User Request forms w/ @lzim 
- [ ] during 1pm HQ meeting: review training_tracker proposal by James & Stacey for training all new team members

### Friday
- [ ] Review MTL facilitate launch meeting notes w/ @ritahitching  and @lijenn 
- [ ] Review GitHub open issues left from James for him to further clarify/work on when return the next week
- Draft further training materials and plan to review with James week of 1/20

### Other
- [ ] Finish inventory of monitor & webcam needs to submit ReqLogic order #890
- [ ] Setup meeting invitation for MTL kickoff calls
- [ ] CFA meeting with David & Jessilyn in Feb
- [ ] Quarterly MTL presentation call - first one end of Feb

## @ritahitching 

**Tuesday Jan 22**
- [ ]  Close out  #853 - waiting on updates to website by James
- [ ] Graphics for GMB / David's paper manuscript (#866) - update provided 
✅   ReqLogic purchase MS Teams & Lindsey account - cancelled - update provided 

**Wednesday Jan 23**
✅   Schedule 1:1 meetings between Lindsey, Stacey, Rita, Jenn in early Feb in preparation for the March Pavir annual review. 
- [ ] Review with Stacey - #825 (SEE & SAY Guides and Master Brochure Comparisons - for EES Elizabeth back 1/15)

**Thursday Jan 24**
- [ ] Pavir - Create list /calendar dates of manual requisitions (#885) -  Create an internal TeamPSD schedule for each vendor that will need manual requisition submissions each year and with due dates for the subscriptions. Perhaps we could create a yearly Outlook reminder that will ping us a few weeks before the deadline as well.
- [ ]  iir_protocol - needs citation keys to be added https://osf.io/p7g8f/wiki/consort_checklist/ 
Comment from Savet: There are locations in this list that mentioned 6months pre-period and other locations that mentions 12 months. Do you have a preference? - NO decision
- [ ] Book TeamPSD dinner with David / Andrea

**Friday Jan 17**
- [ ] Send tt_report via autosend
- [ ] Claire, Gayle and Theresa requested an web camera. 

### Other

- [ ] Send hardcopy materials & swag to facilitators #890
- [ ] VAPOR videos in pre/post emails (issue #870) 
- [ ] VAPOR - Scheduling - Lessons learnt for 3 Veteran engagement groups. 
- Travis Runnels at VAPAHCS.
- Nathaniel Moffet form the Ascend Project in Colorado.
- Leah Wendleton does Veteran-engagement groups for VA researchers.

### Training
- [ ] Finish & get a copy of SOP (OSF Account for online copy of SOP) **DUE: End of 1/15/20**
- [ ] Go through trial/registration/process of putting projects onto OSF (#871) **DUE: 1/17/20**
- [ ] Ensure all Lucid Meeting audios starting from Aug. 2019 are transferred into TeamPSD Meeting Audio folder  **DUE: 1/17/20**
- [ ] VAPOR? **DUE: TBA**
- [ ] (Optional) Reformat Table of Contents for SOP **DUE: TBA**
Deadline for all Training issues: by the end of this week (1/17/20)

### MTL (UI/etc. issues)
- [ ] Fix typos on 'We use cookies' pop-up on MTL UI **DUE: TBA needs training**
- [ ] Fix email signature from MTL (dynamic --> dynamics) **DUE: TBA needs training**
- [ ] Get info./account for mtl.how/test to participate in giving James feedback for any UI related issues **DUE: TBA needs training*
Deadline for all MTL issues: by the end of this week (1/17/20)

**Comment by ritahitching** [comment]
@lzim @staceypark @lijenn 
good morning 😄
I carried over our tasks from last week's HQ thread into this week #895.
I deleted the tasks everyone had crossed out, crossed referenced and closed out issue #889.

**Comment by ritahitching** [comment]
@lzim @staceypark 
Update on Microsoft CRM & Teams (I'll send email update and add screen shot error messages):*

 **ReqLogic**:
- Purchase for single license for Lindsey - cost is $224 per year.  Ready to be submitted, awaiting email acccount details.

**Some issues encountered:**
- Able to download and install Microsoft Teams desktop application on VA machine via the web.
- Account set up not possible using a @va.gov or @pavir.org email address  - only possible with gmail type email account.
- Sign-in blocked by VA browser once noon-va email account has been validated.
- Able to sign up to account using laptop & WiFi. 

@lzim
1. Would you like me to **ask** for the webpage for Microsoft teams to be **unblocked** by contacting ESEDCC.va.gov? 
2. Do you a gmail **account** already set up with Microsoft Teams? If not, I'll create one for you. We will need it to proceed with requisition

thanks!

General update:  **IT Helpdesk Tickets** Installation of Microsoft CRM & Teams - they are ready to come and waiting on licenses from us.

**Comment by ritahitching** [comment]
@lzim  **Update Team PSD dinner:**
Please let me know if you want me to follow up with those that have not responded and if I should book a table. 

**RSVP Availability** 
David & Andrea – not available all week. 
Ash, Stacey, and Jenn available 1/29 & 1/30. 
Rita - 1/30.
Anthony – available 1/29.

**No response** - Swap, Jess, Kathryn, Savet

**Possible Date**: Wednesday 1/ 29 or Thursday 1/30.
**Location**: Milagros in Redwood City

**Comment by lzim** [comment]
Hi HQ! @staceypark @ritahitching @lijenn 

🌻 

Consistent with the new **VISUAL Workflow** for each Workgroup...
Please **organize #hqhuddle  to clarify the following for each day in HQ Workgroup**:
- [ ] HQ **monthly milestones** (to be assigned/visual on GitHub)
- [ ] **Action Items**/Interim Deliverables (to be assigned/visual on Lucid & tt_teampsd report)
- [ ] **Emergent Issues** (aka "sand) that gets "done" (emphasis on the tracking of the done her, not the "to do")

Problems with the approach above: Everyone has daily lists _that continue to mix all three types of things_, which will lead to more "**sand**" completion (i.e., not as important) and less "**milestone**" deliverables (i.e., the most important).
- The biggest problem that I see is that the "Top Priorities" ARE HUGE (the manuscripts are bigger than **milestones** even), would never all be completed this week, and aren't assigned a more manageable **action item** day and with time protected.
- Almost certainly none of the "Top Priorities" will be done this week, given the giant "to do" lists that follow.
- Which is the second problem: Everyone has too much on their list, so they need triage and move things ahead to future weeks to make sure this week's **milestones** and **action items** absolutely get done.
- Third, the list isn't current. For example, the R01 RDIS Progress Report, which we submitted last Thursday is still at the top of Stacey's list. But, it's done. 
- With **a separate issue for each day** we can quickly and easily edit these during huddles throughout the week.
- Finally, the days aren't organized together, they're spread across persons. But, we, **_as a workgroup,_** need to stick the landing each day through effective delegation and prioritization.

Solution
- Create **ONE ISSUE FOR EACH DAY** **with very** brief short lists of **WHO** and **WHAT**, like below. That way, we can double-check them at mid-day #hqhuddles as needed to re-prioritize, triage, etc. 
- And, we will be able to see at a glance when things **should not or can not be delegated** to someone in the workgroup, during a day when they may already have too much "sand" (greater than their 1-2 hours) and/or we need to protect their planned **milestone/action** item time.

### Example: HQ Wednesday Jan 22 
#### Milestones
List milestones that will be completed (if any) this day. List as a check-box and say who will finish it.
- [ ] **Lindsey** presents at HSRD
- [ ] **Lindsey** presents at NIDA

#### Action Items (this should be visible on Lucid Calendar)
List interim deliverables that are due or worked on today. List as a check-box and say who will finish it.
- [ ] **Stacey** begins training of Anthony with video and documentation
- [ ] **Jennifer** uploads the revised SOP with monthly workgroup milestones, etc. described to OSF for review by Stacey & James - attached to **SOP milestone** - Revised SOP would be ready for Mon 1/1 Workgroup Leads and Tues 1/28 All Hands, which means by Friday COB.

#### Emergent Issues (2 hours/day Stacey & Lindsey; 1 hour/day Rita and Jen)
List "sand" that was/will be accomplished during our protected emergent issue time today?
- [ ] **Rita** schedules Feb 1:1 meetings

Thanks, I believe this will be easier to track at a glance in the huddles.
And, I can more efficiently provide feedback about when the priorities are out of order.

Thanks! 👍
